.github/workflows/libjs-test262.yml

```diff
@@ -106,3 +108,32 @@ jobs:
           repository-name: linusg/libjs-website
           token: ${{ secrets.BUGGIEBOT }}
           folder: libjs-website
+
+      - name: Upload per files
+        uses: actions/upload-artifact@v2
+        with:
+          name: 'test262 results'
+          path: libjs-test262/*master.json
```

Comment:
can we make this explicit instead globbing?

Reply:
I incorrectly assumed you had to use a glob to get multiple files